### PR TITLE
Identity Logging iteration - nuke scopes/helpers

### PR DESCRIPTION
**Pull request by HaoK**
Cleanup logging for SignInManager (Will do User/RoleManager) in a follow up PR matching the pattern established here...

cc @divega

### CODE REVIEW

File: src/Microsoft.AspNet.Identity/SignInManager.cs [hunk 1]
```diff
@@ -197,34 +198,32 @@ public virtual void SignOut()
...
-                    if (await UserManager.IsLockedOutAsync(user))
-                    {
-
-                        return Logger.Log(SignInResult.LockedOut);
-                    }
+                    return await LockedOut(user);
                 }
-                return Logger.Log(SignInResult.Failed);
             }
+            // REVIEW: do we want to log normal password failures which would show up here?
```
Comment (divega) [inline src/Microsoft.AspNet.Identity/SignInManager.cs]: Maybe @blowdart will have an opinion. I think it is fine with logging just lockouts.

**Comment by HaoK** [comment]
@blowdart is it ok if we only log Warnings for when users are locked out and don't log anything for normal password mismatch failures?

File: src/Microsoft.AspNet.Identity/SignInManager.cs (hunk 1 above)
Comment (blowdart) [inline src/Microsoft.AspNet.Identity/SignInManager.cs]: Logging normal failures would enable better auditing of potential attacks.

**Comment by divega** [comment]
Nice cleanup! I like that we are not logging everything but only when things go wrong, so we don't need to care that much for the overhead of logging.

:shipit:

**Comment by blowdart** [comment]
Log "normal" mismatches please.

**Comment by HaoK** [comment]
e2d8bfe5cfa8ed014fbce21b924d362ffc87512a

**Comment by HaoK** [comment]
Added logging for password failures, updating rest of managers in the next PR

**Pull request closed without merging**